src/framework/box/Box.spec.jsx

```diff
@@ -0,0 +1,33 @@
+
+import { TestCaseFactory } from 'react-test-kit';
+import Box from './Box.jsx';
+
+describe('Box', () => {
+  describe('Props', () => {
+    describe('classes', () => {
+      it('are added to the element', () => {
+        const props = {
+          classes: 'testClass',
+        };
+        const testCase = TestCaseFactory.create(Box, props);
+        expect(testCase.dom.className).toContain(props.classes);
+      });
+
+      it('doesn\'t replace \'box\' class', () => {
```

Comment:
Plus, we can use _does not_ instead of _doesn't_ to avoid another single quote everywhere else in the same PR.